scripts/ci/run-jenkins.sh

```diff
@@ -83,13 +83,18 @@ if [ -x "/usr/bin/dpkg-architecture" ];
 	wget -qO- https://download.mono-project.com/test/new-certs.tgz| tar zx -C ~/.config/.mono/
 fi
 
+rm -f sdks/Make.config
+
 if [[ ${CI_TAGS} == *'cxx'* ]]; then
 	EXTRA_CONF_FLAGS="$EXTRA_CONF_FLAGS -enable-cxx"
 	MSBUILD_CXX="/p:MONO_COMPILE_AS_CPP=true"
+	echo "ENABLE_CXX=1" >> sdks/Make.config
```

Comment:
Move that to the `products-sdks-ios`, `products-sdks-android` and `webassembly` checks for CI_TAGS.

Reply:
Anyway,  ok.